notebooks/07-Advanced_Topics_and_Plans.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>Divide up and use markdown title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Check to see if I did what you intended.</p>